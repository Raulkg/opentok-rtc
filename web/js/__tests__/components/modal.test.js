var modal = require('../../components/modal.js');
test('snapshot for modal component', () => {
  expect(modal).toMatchSnapshot();
});