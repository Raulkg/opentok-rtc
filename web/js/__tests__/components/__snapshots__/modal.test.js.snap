// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for modal component 1`] = `
Object {
  "Modal": Object {
    "flashMessage": [Function],
    "hide": [Function],
    "show": [Function],
    "showConfirm": [Function],
  },
}
`;
